QueryBuilder.php

```diff
@@ -505,7 +505,8 @@ public function buildFrom($indexes, &$params)
                 }
                 $indexes[$i] = "$index " . $this->db->quoteIndexName($i);
             } elseif (strpos($index, '(') === false) {
-                if (preg_match('/^(.*?)(?i:\s+as|)\s+([^ ]+)$/', $index, $matches)) { // with alias
+                if (preg_match('/^(.*?)(?i:\s+as|)\s+([^ ]+)$/', $index, $matches)) {
+                    // with alias
```

Comment:
I am unsure: this is inline comment for the row of code, so original version should be fine.
@samdark, can you help here, please?